iphone/cli/commands/_build.js

```diff
@@ -6352,7 +6352,24 @@ iOSBuilder.prototype.removeFiles = function removeFiles(next) {
 				// ignore
 			}
 		}, this);
-		done();
+
+		// remove invalid architectures from TitaniumKit.framework for App Store distributions
+		if (this.target === 'dist-appstore') {
+			this.logger.info(__('Removing invalid architectures from TitaniumKit.framework'));
+
+			const titaniumKitPath = path.join(this.buildDir, 'Frameworks', 'TitaniumKit.framework', 'TitaniumKit');
+			async.eachSeries([ 'x86_64', 'i386' ], function (architecture, next) {
+				const args = [ '-remove', architecture, titaniumKitPath, '-o', titaniumKitPath ];
+				this.logger.debug(__('Running: %s', (this.xcodeEnv.executables.lipo + ' ' + args.join(' ')).cyan));
+				appc.subprocess.run(this.xcodeEnv.executables.lipo, args, function (code, out, err) {
```

Comment:
<!--
  0 failure: 
  1 warning:  iphone/cli/comman...
  
  
  DangerID: danger-id-default;
  File: iphone/cli/commands/_build.js;
  Line: 6364;
-->

- :warning: iphone/cli/commands/_build.js line 6364 – 'err' is defined but never used. (no-unused-vars)

Reply:
@garymathews Changes looks good. Should error be checked?